### PR TITLE
Browse videos

### PR DESCRIPTION
**Pull request by Moutasim02**

### CODE REVIEW

**Comment by binHarby** [comment]
Hmm this one is bit complex, its a new branch based on the LoginFix branch which has not been merged yet. Its like two pull request entangled together. 

I'll wait for you to finish the LoginFix branch, then I'll merge that branch.

once that LoginFix branch is merged, get back to this branch, and sync it with master so it isn't behind

**Comment by Moutasim02** [comment]
I see, I think this happened because I didn't go back to the master branch once I finished from login, so I checked out through the Login branch. 

Anyways I will do it as suggested.

**Review by binHarby — APPROVED** [review]
Adding Browse Videos Initial and Bootstrap link tags. Approved

**Pull request merged**
